### DISCUSSION TITLE
Hourly fire weather

### DISCUSSION THREAD

**Discussion by paulnadler** [weather-gov/api#743] [General]
In the MapClick web API there are hourly fire weather parameters such as lightning activity level, which are not present here and were also never available in the old MapClick hourlyDWML or JSON. 

Are there any plans to add these to /gridpoints or /hourly?

**Comment by dwhitemv25** [comment]
Those elements will only be present in `/gridpoints` if your WFO sends them. For my gridpoint, `lightningActivityLevel` and `hainesIndex` are there. 

Considering that there is a PNS on [removing Haines index and LAL from the fire weather forecast products](https://www.weather.gov/media/notification/pdf_2023_24/pns24-38_proposed_discontinuation_of_haines_index_and_lal.pdf) citing obsolescence, I don't know if there is much priority to keep those particular elements alive.

**Reply by paulnadler** [reply]
Thanks for the info. I didn't see that.
